docs: add encryption and upload sections (#136)
sdsc-ordes/modos-api: docs/tutorials/genomics_streaming.md

# Load MODO from remote storage
modo=MODO(path= 's3://modos-demo/ex', endpoint = 'http://localhost')

> # Stream a specific regiontwo-fold: the data itself is encrypted, and so is the mechanism for unlocking it.

Reviewer: **comment:** I think this was meant to go elsewhere

:::{warning}
We highly recommend using the `MODOs` CLI for streaming. The output can directly be passed to tools like <a href="https://www.htslib.org/" target="_blank">samtools</a>. Streaming using the `MODOs` python api will return a <a href="https://pysam.readthedocs.io/en/stable/" target="_blank">pysam</a> object. `pysam` does not allow reading from byte-streams and thus the streamed region will be written into an temporary file before parsing to `pysam`. For large files/regions this can cause issues.
:::

## Data encryption and decryption

> Genomic data is usually sensitive and especially data sharing poses the risk for data security.

Reviewer: ```suggestion
Genomic data is typically sensitive, and data sharing increases the risk to data security.
```
**suggestion:** rephrase

> `MODOs` supports <a href="https://samtools.github.io/hts-specs/crypt4gh.pdf" target="_blank">Crypt4GH</a> an encryption format developed by the Global Alliance for Genomics and Health (GA4GH).

Reviewer: ```suggestion
`MODOs` supports <a href="https://samtools.github.io/hts-specs/crypt4gh.pdf" target="_blank">Crypt4GH</a>, an encryption format developed by the Global Alliance for Genomics and Health (GA4GH).
```

**suggestion:** grammar

Crypt4GH encryption is based on authenticated envelope encryption that will encrypt the data itself as well as the key to decrypt the data.

> In `MODOs` all genomic files can be encrypted or decrypted with one command call:

Reviewer: ```suggestion
In `MODOs`, all genomic files can be encrypted or decrypted with a single command call:
```
**suggestion:** grammar
